lib/errors.d.ts

```diff
@@ -33,13 +33,15 @@ export declare class NoLivingConnectionsError extends ElasticsearchClientError {
 export declare class SerializationError extends ElasticsearchClientError {
   name: string;
   message: string;
-  constructor(message: string);
+  data: object;
+  constructor(message: string, data: object);
 }
 
 export declare class DeserializationError extends ElasticsearchClientError {
   name: string;
   message: string;
-  constructor(message: string);
+  data: object;
```

Comment:
This should be a string.

```suggestion
  data: string;
```